src/java.base/share/classes/java/lang/AutoCloseable.java

```diff
@@ -51,7 +52,8 @@
  * @author Josh Bloch
  * @since 1.7
  */
-@AnnotatedFor({"lock"})
+@AnnotatedFor({"lock", "mustcall", "nullness"})
+@InheritableMustCall("close")
```

Comment:
With this change, we should also remove the `@InheritableMustCall("close")` annotation on `java.io.Closeable` (since it extends `AutoCloseable`).

Reply:
I had thought of this but left it for emphasis.  Now I have removed it.